src/common/syncjournaldb.cpp

```diff
@@ -408,6 +408,12 @@ bool SyncJournalDb::checkConnect()
         return sqlFail(QStringLiteral("Create table metadata"), createQuery);
     }
 
+    createQuery.prepare("CREATE TABLE IF NOT EXISTS last_sync(time INTEGER(8));");
```

Comment:
can you modify this table such that we can use it to log more than just last sync time but any information we could use ?
Something like `sync_status` with a column `name` and `time`

Reply:
Sorry, I was confused when answering.
I really meant something that could store keys and values. So column names could be `name` and `value`.

Reply:
@mgallien is it now like you thought it?